app/src/main/java/com/peacecorps/pcsa/safety_tools/FormattedSafetyPlanBasicsContentFragment.java

```diff
@@ -0,0 +1,90 @@
+package com.peacecorps.pcsa.safety_tools;
+
+import android.graphics.Typeface;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentManager;
+import android.text.Html;
+import android.text.style.RelativeSizeSpan;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.bluejamesbond.text.DocumentView;
+import com.bluejamesbond.text.hyphen.DefaultHyphenator;
+import com.bluejamesbond.text.style.JustifiedSpan;
+import com.bluejamesbond.text.style.TextAlignment;
+import com.bluejamesbond.text.util.ArticleBuilder;
+import com.peacecorps.pcsa.JustificationUtil;
+import com.peacecorps.pcsa.R;
+
+/**
+ * One Fragment which acts like a placeholder for every different screens of Safety Plan Basics
+ * Created by ashu on 5/2/17.
+ */
+
+public class FormattedSafetyPlanBasicsContentFragment extends DialogFragment {
```

Comment:
No need to create a formatted fragment. Is there any dependency for not changing the original fragment?